packages/renderer/MdToHtml.ts

```diff
@@ -48,6 +48,8 @@ const rules: RendererRules = {
 	code_inline: require('./MdToHtml/rules/code_inline').default,
 	fountain: require('./MdToHtml/rules/fountain').default,
 	mermaid: require('./MdToHtml/rules/mermaid').default,
+	table_open: require('./MdToHtml/rules/table_open').default,
+	table_close: require('./MdToHtml/rules/table_close').default,
```

Comment:
Since these two functions work together, could you put them together in the same file and use a meaningful name please? The basic structure would be like this:

```ts
export default {
	plugin: (markdownIt: any, params: any) => {
		markdownIt.renderer.rules.table_close = function(tokens: any[], idx: number, options: any, _env: any, self: any) {
			// ...			
		};

		markdownIt.renderer.rules.table_open = function(tokens: any[], idx: number, options: any, _env: any, self: any) {
			// ...
		};
	},
};
```

What does the plugin do exactly? It makes the table horizontally scrollable on mobile, is that correct?

Reply:
Yes, this plugin makes the table horizontally scrollable on mobile, so the maximum width of the table is the screen width. When dragging notes down, the interface does not sway left or right.

```
<div class="joplin-table-div" >
      <table class="maps-to-line" source-line="186" source-line-end="312">
     .....
      </table>
</div>
```

Reply:
> ```ts
> export default {
> 	plugin: (markdownIt: any, params: any) => {
> 		markdownIt.renderer.rules.table_close = function(tokens: any[], idx: number, options: any, _env: any, self: any) {
> 			// ...			
> 		};
> 
> 		markdownIt.renderer.rules.table_open = function(tokens: any[], idx: number, options: any, _env: any, self: any) {
> 			// ...
> 		};
> 	},
> };
> ```

 Rebuild to: packages/renderer/MdToHtml/rules/tableHorizontallyScrollable.ts

Reply:
Do you agree with the option name "tableHorizontalyScrollable"? If you agree, I will try to commit modifications of 
 introduction options(readme/apps/markdown.md) and translations file(packages/tools/locales)